record_web/lib/record_web.dart

```diff
@@ -187,4 +188,25 @@ class RecordPluginWeb extends RecordPlatform {
   void _log(dynamic msg) {
     if (kDebugMode) print(msg);
   }
+  
+  @override
```

Comment:
This method does nothing.
Please filter on, audio input devices and fill the resulting collection.